src/types/IJaaSMeetingProps.ts

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+
+/* eslint-disable-next-line */
+import { JaaSMeeting } from '..';
+import IJitsiMeetExternalApi from './IJitsiMeetExternalApi';
+
+/**
+ * The type of the React {@code Component} props of {@link JaaSMeeting}.
+ */
+export default interface IJaaSMeetingProps {
```

Comment:
THis looks 99% similar to the other one, maybe we can define a base interface from which both inherit?